task_sdk/src/airflow/sdk/api/client.py

```diff
@@ -157,6 +157,12 @@ def get(self, key: str) -> VariableResponse:
         resp = self.client.get(f"variables/{key}")
         return VariableResponse.model_validate_json(resp.read())
 
+    def put(self, msg):
+        """Put a variable through the API server."""
+        key = msg.key
+        put_resp = self.client.put(f"variables/{key}", content=msg.model_dump_json())
+        return put_resp
```

Comment:
This should probably not have a return (raise on error only)

Reply:
If the execution API complains with an error, this will probably handle that without the return, correct?

Reply:
In that case, we can just make do without caring about the response

Reply:
So i mean to say, if i remove the part where the client returns the error, if there is an error, it will anyways propagate down to the supervisor. The error is a standard: `{"message": "Variable successfully set"}` anyways. WDYT?

Reply:
Yeah, we can do that. That will reduce the coupling and this will generally serve as a pattern for all the "setters" in the client.

Reply:
I added {"ok": True} logic as mentioned above in the last commit